blob/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@azure/storage-blob",
   "version": "10.0.0-preview",
   "description": "Microsoft Azure Storage SDK for JavaScript - Blob",
-  "main": "./dist/lib/index.js",
-  "module": "./es6dist/index.js",
+  "main": "./dist/index.js",
+  "module": "./dist-esm/lib/index.js",
   "browser": {
```

Comment:
Should update the browser filed accordingly

`"./dist/index.browser.js"` is the rollup build for browser. Please refer to another comment for the reasons why we need separate 2 builds for Node.js and browsers.

```
"browser": {
   "./dist/index.js": "./dist/index.browser.js",
  "./dist-esm/lib/index.js": "./dist-esm/lib/index.browsers.js"
  "os": false,
  ...
}
```